lib/kitchen/directions/bake_numbered_exercise/main.rb

```diff
@@ -3,8 +3,9 @@
 module Kitchen
   module Directions
     module BakeNumberedExercise
-      def self.v1(exercise:, number:, suppress_solution: false)
-        V1.new.bake(exercise: exercise, number: number, suppress_solution: suppress_solution)
+      def self.v1(exercise:, number:, suppress_solution_if: false, note_suppressed_solutions: false)
```

Comment:
@brittany-johnson I have left it as JP created it, because it can be either true, false or `:even?` passing as a symbol. See my comment here: https://github.com/openstax/kitchen/pull/181/files#r647324906